test/hotspot/gtest/gtestMain.cpp

```diff
@@ -112,27 +110,37 @@ class JVMInitializerListener : public ::testing::EmptyTestEventListener {
   int _argc;
   char** _argv;
   bool _is_initialized;
+  JavaVM* jvm;
```

Comment:
underscore missing?

Reply:
and `_is_initialized` is now redundant and could be aliased to `_jvm != NULL`

Reply:
Yes good point - fixed.